pages/index.tsx

```diff
@@ -49,7 +52,9 @@ export default function Home() {
           invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et
           accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata
           sanctus est Lorem ipsum dolor sit amet.
-          <Button className="ml-[38px] mt-1">read more</Button>
+          <Button className="ml-[38px] mt-1">
+            <Link href="/why-bluebird">read more</Link>
```

Comment:
I think the page is supposed to be /why not /why-bluebird. Also please merge in the latest main, since it has the why bluebird page added so then it can be tested